src/main/webapp/index.html

```diff
@@ -34,5 +34,8 @@ <h1>CodeU Starter Project</h1>
     their URL.</p>
     <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
     You could start by modifying this page to tell the world more about your team.</p>
+
```

Comment:
I believe this is an unrelated change; please make it in a different pull request.

Reply:
How do I take this out of this request, and put it in a new one?